### ISSUE TITLE
UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 20): Error: Error in file: E:/path/xxx.jpg

### ISSUE THREAD

**Issue by summit2020** [imagemin/imagemin-mozjpeg#24]
Throwing exception for few images ?

UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 20): Error: Error in file: E:/path/xxx.jpg

**Comment by usamamashkoor** [comment]
Facing the same issue did you find any solution...?

**Comment by summit2020** [comment]
Yeah am facing the same issue & dint yet find out the cause of an issue .. please help me out of this ..
Thanks

**Comment by usamamashkoor** [comment]
Use imagemin-mozjpeg 6.0.0 version instead of 7.0.0 

Maybe it fix the issue for you.

**Comment by usamamashkoor** [comment]
also use catch block after the `.then ` block.
